app/models/sms_broker/outgoing_message.rb

```diff
@@ -17,7 +17,7 @@ class OutgoingMessage < ActiveRecord::Base
 
     @@after_create_hooks = Array.new
 
-    # :reek:Manual-Dispatch
+    # :reek:ManualDispatch
```

Comment:
this change is not really connected to the issue, but we had a warning saying "there is no such cop", cause it was wrong-named 🍳